### DISCUSSION TITLE
Union subtypes with conflicting field value type

### DISCUSSION THREAD

**Discussion by danielcondemarin** [kamilkisiela/graphql-inspector#2659] [Q&A]
Is it possible to validate the following situation as part of the `diff` command?

https://github.com/dimaMachina/graphql-eslint/issues/2008#issue-1993131980

**Comment by dimaMachina** [comment]
Hello, <s>what’s `diff` command?</s> Ah I was thinking this is graphql-eslint discussions

**Reply by danielcondemarin** [reply]
https://the-guild.dev/graphql/inspector/docs/commands/diff
